src/routes/requestRoute.js

```diff
@@ -12,6 +12,11 @@ const requestRoutes = app => {
     Authorization.checkToken,
     Validate.validateUserInput,
     RequestController.bookAReturnTrip);
+
+  app.patch('/api/v1/request/reject/:id',
```

Comment:
We don't pass ID as a URL parameter. Pass it in the request body

Reply:
The ID has to be passed into the request parameter so it can be easily fetched when searching for a particular request.

Reply:
You can check the code base for previous routes. All id's were passed in the request body for security reasons. you can confirm from the TTL who effected the changes

Reply:
Okay. Thanks for the feedabck